signin.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="ko">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>로그인</title>
+    <link rel="stylesheet" href="css/signin_style.css">
+  </head>
+  <body>
+    <div class="content">
+      <div>
+        <a class="logo" href="index.html"><img alt="logo" src="images/logo.svg"></a>
+        <div class="sign-up">
+          회원이 아니신가요?
+          <a href="/signup.html">회원 가입하기</a>
+        </div>
+      </div>
+      <form>
+        <div class="input-email">
+          <label for="email">이메일</label>
+          <input id="email" name="email" type="email" placeholder="e-mail">
+        </div>
+        <div class="input-password">
+          <label for="password">비밀번호</label>
+          <div class="checkbox">
+            <input id="password" name="password" type="password" placeholder="비밀번호">
+            <button><img alt="blind" src="images/eye-on.svg"></button>
```

Comment:
button 태그 사용시 type을 명시해주는 것이 좋습니다. 해당 버튼의 경우 클릭시 비밀번호가 보이고 안 보이고하는 역할이니 type="button" 이 적절할 것 같네요.
```suggestion
            <button type='button'><img alt="blind" src="images/eye-on.svg"></button>
```